clusterloader2/testing/watch-list/job.yaml

```diff
@@ -22,5 +22,5 @@ spec:
               cpu: "6"
           command: [ "watch-list" ]
           # TODO(p0lyn0mial): bring back 16 informers
-          args: [ "--alsologtostderr=true", "--v=4", "--timeout={{.Duration}}", "--count=8", "--namespace=watch-list-1", "--enableWatchListFeature={{.EnableWatchListFeature}}"]
+          args: [ "--alsologtostderr=true", "--v=4", "--timeout={{.Duration}}", "--count=6", "--namespace=watch-list-1", "--enableWatchListFeature={{.EnableWatchListFeature}}"]
```

Comment:
nit, we should expand the comment to explain this magic number, is going to be hard to trace back from the git logs

Reply:
This is just temporary to gather some data. I am going to bring back the previous value once we finish with the experiment.